src/components/PersonCards.js

```diff
@@ -41,8 +38,9 @@ export const PersonCard = ({ person, menu, subtitle, slug }) => {
     </A>
     <div className='person-body'>
       <A className='name' to={`/u/${id}`}>{name}</A>
-      <div className='role'>{role_and_age}</div>
-      {subtitle && <div className='subtitle'>{subtitle}</div>}
+      <div className='role'>
+        {(isModerator ? 'Moderator' : 'Member') + ` since ${humanDate(joined_at)}`}
```

Comment:
lol.
got it

On Fri, May 13, 2016 at 3:06 PM, Lawrence Wang <notifications@github.com>
wrote:

> In src/components/PersonCards.js
> <https://github.com/Hylozoic/hylo-redux/pull/48#discussion_r63257645>:
>
> > @@ -41,8 +38,9 @@ export const PersonCard = ({ person, menu, subtitle, slug }) => {
> >      </A>
> >      <div className='person-body'>
> >        <A className='name' to={`/u/${id}`}>{name}</A>
> > -      <div className='role'>{role_and_age}</div>
> > -      {subtitle && <div className='subtitle'>{subtitle}</div>}
> > +      <div className='role'>
> > +        {(isModerator ? 'Moderator' : 'Member') + ` since ${humanDate(joined_at)}`}
>
> i meant this:
>
> <div className='role'>
>   {isModerator ? 'Moderator' : 'Member'} since {humanDate(joinedAt)}
> </div>
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/Hylozoic/hylo-redux/pull/48/files/25c8fb27d2493613b9d37d4c534746911bae10f7..2df60bbc95537e27803ceda6517d694cc8a60b26#r63257645>
>



-- 
Robbie Carlton

719.358.3482
<robbie@authenticmanprogram.com>